std/net/curl.d

```diff
@@ -1510,12 +1511,8 @@ private mixin template WorkerThreadProtocol(Unit, alias units)
     }
 }
 
-// Workaround bug #2458
```

Comment:
Ok, the comment can be removed and this can be moved inside when https://issues.dlang.org/show_bug.cgi?id=15831 is fixed.

Maybe add this note:
```
// @@@@BUG 15831@@@@
// this should be inside byLineAsync
```